rewrite-xml/src/main/java/org/openrewrite/xml/trait/SpringXmlReference.java

```diff
@@ -31,55 +28,9 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
-@Value
-class SpringReference implements Reference {
-    Cursor cursor;
-    Kind kind;
+public class SpringXmlReference {
```

Comment:
I think I should extend the `XmlReference` here.

Reply:
I was in doubt about this because in this case the XmlReference and SpringXmlReference perfectly fit eachother. When you run into a XmlReference that needs a different implementation of `getValue` it would make sense to introduce this inheritance. So I agree we might as well start now :)